Cargo.toml

```diff
@@ -29,5 +29,7 @@ missing_errors_doc = "allow"
 module_name_repetitions = "allow"
 
 [workspace.dependencies]
+thiserror = "1"
+sha2 = "0.10"
```

Comment:
Dependabot doesn't handle updating workspace dependencies properly - so anything that needs updating (ie: anything that isn't a dependency on another crate in this repo) can't be listed here for now (hence #614). 

Longer term perhaps we can submit a PR to https://github.com/dependabot/dependabot-core, but for now can you move these to the individual crates?